hardhat.config.ts

```diff
@@ -0,0 +1,118 @@
+import * as dotenv from "dotenv";
+import { task } from "hardhat/config";
+import { HardhatUserConfig } from 'hardhat/types';
+import 'hardhat-deploy';
+import "hardhat-gas-reporter";
+import "solidity-coverage";
+import "@nomiclabs/hardhat-ethers";
+import "hardhat-erc1820";
+
+import {
+  RSK_TEST_NET_CHAIN_ID,
+  RSK_MAIN_NET_CHAIN_ID,
+  KOVAN_TEST_NET_CHAIN_ID
+} from "./utils/chains";
+
+import "@openzeppelin/hardhat-upgrades";
+
+dotenv.config();
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+// You need to export an object to set up your config
+// Go to https://hardhat.org/config/ to learn more
+
+const ACCOUNTS = {
+  count: 10,
```

Comment:
this will generate 10 address, i think we only use 1 for the deployment

Reply:
Done at https://github.com/rsksmart/swap-rbtc/pull/1/commits/f99d281695b2c0d76f0de010564d8a1952ed7b53